react-game/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
please don't push package-lock folder (add it to git ignore file)

Reply:
done